docs/querying/basics.md

```diff
@@ -116,6 +116,10 @@ The following expression selects all metrics that have a name starting with `job
 
     {__name__=~"job:.*"}
 
+All label matchers are case-sensitive. To perform a case-insensitive match, use a 
+regex with the `(?i)` option at the beginning. Partial case-insensivity can be 
+achieved with the `(?i:pattern)` group.  
```

Comment:
You could add examples as for the other paragraphs.

```
{method=~"(?i)get"} # case-insensitive match on "get"
{method=~"(?i:g)et"} # match on "get" or "Get"
```

Reply:
I'm not sure it's appropriate to add niche details on how regexes work, particularly as we don't even cover the basic syntax here. I think this is already covered by the link just below.